app/src/main/res/values/strings.xml

```diff
@@ -50,6 +50,11 @@
     <string name = "more_info">More Info</string>
     <!--Login-->
     <string name = "sign_in_google">Sign in with Google</string>
+    <string name="login_swipe_text">Swipe to learn more…</string>
+    <string name="login_activities_text">Lasta allows you to find nearby outdoor activities and track your performances.</string>
+    <string name="login_friends_text">You are more of a social sportsman ? We got you covered ! Invite and add your friends in the app and watch their latest activities.</string>
+    <string name="login_free_text">The best part ? It is totally free !
+Try it now with a simple Google sign in and join our fanstastic sports addict community.</string>
```

Comment:
I wouldn't say "try it now with a simple google sign in" rather "Sign in to join our fantastic sports addict community!" I don't think mentioning google is that relevant in the text for the user (mostly a dev part on our side). Also make sure to change it for the other languages too !

Reply:
You are totally right, I will change this